### PR TITLE
Publicar Introdução aos Dados Abertos Conectados

### PR DESCRIPTION
**Pull request by DanielAlvesLABDH**
Publicar a tradução de "Introduction to the Principles of Linked Open Data"

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by DanielAlvesLABDH** [comment]
Olá @ericbrasiln. Quando tiver disponibilidade, agradeço o seu olhar atento numa última revisão deste PR. Será a 33!!!

**Comment by DanielAlvesLABDH** [comment]
@ericbrasiln and @anisa-hawes can you help identify the origin of the error in the build? I see that it is on the images, but can't figured out what is wrong :-(

**Comment by anisa-hawes** [comment]
Hello @DanielAlvesLABDH. I think the problem was that the `original:` field was missing from the YAML. This means that the file doesn't include instructions for where to locate the image files.

I might also need to fix the broken link to <http://dilipad.history.ac.uk/> which is the project website for Digging Into Linked Parliamentary Data. Perhaps we could exchange it for a link to the paper's final report <https://repository.jisc.ac.uk/6544/1/DiLiPaD_final_report_1.pdf>. 

I have a strange feeling that I've come across this broken link before...

Update: Yep. It has been updated in /en/lessons/intro-to-linked-data & /es/lecciones/introduccion-datos-abiertos-enlazados already.

**Comment by DanielAlvesLABDH** [comment]
Thank you @anisa-hawes, that was the problem. As for the link, I think your suggestion could be a good alternative.

**Comment by anisa-hawes** [comment]
Ideally, I'd also like us to take the opportunity to add in descriptive alt-text in for each image. I would need your help (or @ericbrasiln's, if he has time?) to do this, because the alt-text will need to be written in Portuguese (obviously). I've collated some links to guidance for writing alt-text on our Wiki page: [Achieving Accessibility: Alt text, Colour, Contrast](https://github.com/programminghistorian/jekyll/wiki/Achieving-Accessibility-Alt-text-Colour-Contrast), although we would very much welcome your suggestions for resources in PT, if you know of anything useful?

In terms of syntax, `alt=`  is slotted in between the filename and caption using this format: `{% include figure.html filename="file-name.png" alt="Visual description of figure image" caption="Caption text to display" %}`

Meanwhile, I can double-check the typesetting and add in perma.cc links where appropriate 🙂

File: pt/licoes/introducao-dados-abertos-conectados.md [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+6. Outras leituras e recursos
+
+A conclusão deste tutorial poderá levar algumas horas e poderá ser útil reler algumas secções para solidificar a sua compreensão. Os termos técnicos foram ligados à sua página correspondente na Wikipedia e encoraja-se a que faça uma pausa e leia sobre termos que considere desafiadores. Depois de ter aprendido alguns dos princípios-chave do LOD, a melhor maneira de melhorar e solidificar esse conhecimento é praticar. Este tutorial fornece oportunidades para fazê-lo. No final da lição, deverá compreender os princípios básicos de LOD, incluindo termos e conceitos-chave.
+
+Se precisar aprender a como explorar LOD usando a linguagem de consulta [SPARQL](https://pt.wikipedia.org/wiki/SPARQL), recomenda-se a lição de Matthew Lincoln ['*Using SPARQL to access Linked Open Data*'](/en/lessons/retired/graph-databases-and-SPARQL) (em inglês) (Nota: a lição deste link encontra-se desatualizada e já não é mantida pelo _Programming Historian_. Por favor veja a nota inicial dessa página sobre a razaão dessa lição ter sido retirada), que segue praticamente a visão geral fornecida nesta lição.
```
Comment (ericbrasiln) [inline pt/licoes/introducao-dados-abertos-conectados.md]: corrigir 'razaão' para 'razão'

File: pt/licoes/introducao-dados-abertos-conectados.md [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
... `history.ac.uk`, houve uma separação clara entre URIs e as páginas do website. Por exemplo, um dos URIs do projeto Tobias era 'http://data.history.ac.uk/tobias-project/person/15601'. Embora o formato dos URIs acima mencionados seja o mesmo que um URL, eles não se ligam a websites (tente colá-lo num navegador web). Muitas pessoas novas no LOD acham isto confuso. Todos os URLs são URIs, mas nem todos os URIs são URLs. Um URI pode descrever qualquer coisa, enquanto o URL descreve a localização de algo na web. Assim, um URL diz-lhe a localização de uma página web, de um ficheiro ou algo semelhante. Um URI faz apenas o trabalho de identificar algo. Tal como o Número internacional Normalizado do Livro (International Standard Book Number ou [ISBN](https://www.iso.org/standard/36563.html) (em inglês) `978-0-1-873354-6` identifica exclusivamente uma edição de capa dura de _Baptism, Brotherhood and Belief in Reformation Germany_, de Kat Hill, mas não diz onde obter uma cópia. Para isso precisaria de algo como um [número de acesso](https://pt.wikipedia.org/wiki/N%C3%BAmero_de_acesso_(biblioteconomia)), que lhe dá uma localização exata de um livro numa prateleira de uma biblioteca específica.
```
Comment (ericbrasiln) [inline pt/licoes/introducao-dados-abertos-conectados.md]: O link para o projeto Tobias [http://www.history.ac.uk/projects/digital/tobias](http://www.history.ac.uk/projects/digital/tobias)não abre o que se espere, mas sim a página [https://www.history.ac.uk/research/digital-history](https://www.history.ac.uk/research/digital-history)

File: pt/licoes/introducao-dados-abertos-conectados.md [hunk 3]
```diff
@@ -0,0 +1,419 @@
...
+
+Poderíamos igualmente ter criado as nossas triplas semânticas desta forma:
+
+    "Charles Rosen" aprendeuPianoCom "Moriz Rosenthal" .
+    "Moriz Rosenthal" aprendeuPianoCom "Franz Liszt" .
+
+Estamos a inventar exemplos simplesmente para fins de ilustração, mas se quiser ligar os seus dados a outros datasets na "nuvem de dados conectados" deve olhar para as convenções que são utilizadas nesses datasets e fazer o mesmo. Na verdade, esta é uma das características mais úteis de LOD porque muito do trabalho já foi feito. As pessoas têm passado muito tempo a desenvolver formas de modelar a informação dentro de uma determinada área de estudo e a pensar sobre como as relações dentro dessa área podem ser representadas. Estes modelos são geralmente conhecidos como ontologias. Uma ontologia é uma abstração que permite a representação de um conhecimento particular sobre o mundo. Neste sentido, estas são bastante recentes e foram concebidas para fazer o que uma [taxonomia](https://pt.wikipedia.org/wiki/Taxonomia_(geral)) hieráquica faz (pense na classificação das espécies na [Taxonomia de Lineu](https://pt.wikipedia.org/wiki/Taxonomia_de_Lineu), mas de uma forma mais flexível.
```
Comment (ericbrasiln) [inline pt/licoes/introducao-dados-abertos-conectados.md]: corrigir 'hieráquica' para 'hierárquica'

File: pt/licoes/introducao-dados-abertos-conectados.md [hunk 4]
```diff
@@ -0,0 +1,419 @@
...
+
+A linguagem de consulta que usamos para LOD é chamada [SPARQL](https://pt.wikipedia.org/wiki/SPARQL). É um daqueles acrónimos recursivos amados pelos técnicos: ***S**PARQL **P**rotocol **a**nd **R**DF **Q**uery **L**anguage* (Protocolo SPARQL e Linguagem de Consulta).
+
+Como mencionado no início, o *Programming Historian* tem [uma lição completa](/en/lessons/retired/graph-databases-and-SPARQL) (em inglês), de Matthew Lincoln, sobre a utilização do SPARQL (embora não seja já mantida (ver nota no início desta tradução). A secção final aqui presente é apenas uma visão geral dos conceitos básicos. Se o SPARQL despertar o seu interesse, pode obter uma fundamentação completa no tutorial de Lincoln.
+
+Vamos realizar as nossas consultas SPARQL na [DBpedia](https://www.dbpedia.org/), que é um enorme conjunto de LOD derivado da Wikipedia. Além de estar cheio de informação que é muito difícil de encontrar através da habitual interface da Wikipédia, tem vários "pontos de extremindade" (end points) SPARQL - interfaces onde se podem digitar as consultas SPARQL e obter resultados a partir das triplas semânticas da DBpedia.
```
Comment (ericbrasiln) [inline pt/licoes/introducao-dados-abertos-conectados.md]: corrigir 'extremindade' para 'extremidade'

File: pt/licoes/introducao-dados-abertos-conectados.md [hunk 5]
```diff
@@ -0,0 +1,419 @@
...
+* [*Linked Data guides and tutorials*](http://linkeddata.org/guides-and-tutorials) (em inglês)
+* Dominic Oldman, Martin Doerr e Stefan Gradmann, '*Zen and the Art of Linked Data: New Strategies for a Semantic Web of Humanist Knowledge*', em *A New Companion to Digital Humanities*, editado por Susan Schreibman et al.
+* Max Schmachtenberg, Christian Bizer e Heiko Paulheim, [*State of the LOD Cloud 2017*](http://linkeddatacatalog.dws.informatik.uni-mannheim.de/state/) (em inglês)
+* David Wood, Marsha Zaidman e Luke Ruth, *Linked Data: Structured data on the Web*, Manning, 2014
+
+## Agradecimentos
+
+Gostaria de agradecer aos meus dois colegas revisores, Matthew Lincoln e Terhi Nurmikko-Fuller e ao meu editor, Adam Crymble, por me ajudarem generosamente a melhorar esta lição com numerosas sugestões, esclarecimentos e correções. Este tutorial baseia-se num outro escrito como parte do '*Thesaurus of British and Irish History as SKOS*' [*(Tobias) project*](http://www.history.ac.uk/projects/digital/tobias) (em inglês), financiado pelo [AHRC](http://www.ahrc.ac.uk/) (em inglês). A lição foi revista para o projeto *Programming Historian*.
```
Comment (ericbrasiln) [inline pt/licoes/introducao-dados-abertos-conectados.md]: Mesmo coisa do meu comentário da linha 144.

**Review by ericbrasiln — CHANGES_REQUESTED** [review]
@DanielAlvesLABDH I've made some corrections, mostly orthographic, and indicated some wrong links. 

@anisa-hawes I guess all images has `alt=text` now.

File: pt/licoes/introducao-dados-abertos-conectados.md (hunk 2 above)
Comment (DanielAlvesLABDH) [reply pt/licoes/introducao-dados-abertos-conectados.md]: @anisa-hawes do you think we can change that link for this one: https://blog.history.ac.uk/2015/10/representing-tobias-our-new-ahrc-funded-project-building-on-the-bibliography-of-british-and-irish-history/

**Comment by DanielAlvesLABDH** [comment]
Thank you @ericbrasiln. The one thing that it's missing is to correct the link for the Tobias project. Let's see what @anisa-hawes says...

File: pt/licoes/introducao-dados-abertos-conectados.md (hunk 2 above)
Comment (anisa-hawes) [reply pt/licoes/introducao-dados-abertos-conectados.md]: Perhaps we could replace the link on l.144 with this one: <https://gtr.ukri.org/projects?ref=AH%2FN003446%2F1#/tabOverview> which provides an overview of the Tobias Project. I can't locate the relevant page on IHR's website...

**Comment by anisa-hawes** [comment]
Thank you @ericbrasiln and @DanielAlvesLABDH. Perhaps we could replace the link on l.144 with this one: <https://gtr.ukri.org/projects?ref=AH%2FN003446%2F1#/tabOverview> which provides an overview of the Tobias Project. 

I can't locate the relevant page on IHR's website...


(Apologies, as I haven't yet had time to perma.cc the external links in this lesson.)

File: pt/licoes/introducao-dados-abertos-conectados.md (hunk 2 above)
Comment (DanielAlvesLABDH) [reply pt/licoes/introducao-dados-abertos-conectados.md]: I agree. Will replace the link in the file

**Comment by DanielAlvesLABDH** [comment]
No worries @anisa-hawes. Thanks for the help. I think we can make that change latter. @ericbrasiln all your suggested changes are introduced. I think we can close this. Thanks

**Review by ericbrasiln — APPROVED** [review]

**Comment by DanielAlvesLABDH** [comment]
@anisa-hawes the PR went well, we closed it about 6 hours ago, but the lesson still doesn't show on the live site. What could have happen?

**Comment by anisa-hawes** [comment]
Dear @DanielAlvesLABDH,

I think an update may be required in our build workflow, so I have opened an Issue #2761 to investigate.

In this case, I've been able to re-run the job, and [it has merged in successfully](https://programminghistorian.org/pt/licoes/introducao-dados-abertos-conectados) 😄 

Making a note here to let @GeeAlmeida know that the doi for this new lesson isn't live yet, but will be soon so we can celebrate on Twitter later!

**Comment by DanielAlvesLABDH** [comment]
Many thanks @anisa-hawes. It worked. It's now live. 33!

**Pull request merged**